src/Codeception/Test/Unit.php

```diff
@@ -75,6 +75,25 @@ protected function _after()
     }
 
     /**
+     * If the method exists (PHPUnit 5) forward the call to the parent class, otherwise
+     * call `expectException` instead (PHPUnit 6)
+     */
+    public function setExpectedException($exception, $message = '', $code = null)
+    {
+        if (method_exists($this, 'expectException')) {
+            $this->expectException($exception);
```

Comment:
@MontealegreLuis I think that the warning should be shown if `expectException` exists.